contents/news/tips-on-how-to-prepare-a-talk/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Tips on preparing a talk
+date: 2015-03-20
+template: article.jade
+category: news
+---
+
+So you decided that you want to visit the JSunconf this year. As you might know a unconference lives from the visitors and he Talks are prepared by the visitors and get choosen in the morning of each conference day by the crowd. This gives everyone the chance to prepare an awesome talk. This is where you all come in.
```

Comment:
`This gives everyone the chance to prepare an awesome talk` maybe change to `This gives everyone the chance to give an awesome talk`